recipes/python-javabridge/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "python-javabridge" %}
+{% set version = "4.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 9fd9c85e6a32f4edd9fff659d387cbf4c1c91fde9fe3670e4f6d37d79eb6200f
+
+build:
+  number: 0
+  script: '{{ PYTHON }} -m pip install --no-deps --ignore-installed . '
+  skip: true  # [win]
+
+requirements:
+  build:
+    {{ compiler('c') }}
+  host:
+    - numpy >=1.18.2
+    - cython >=0.29.16
+    - openjdk
+    - pip
+    - python
+    - setuptools
+  run:
+    - numpy >=1.18.2
```

Comment:
Does this package builds against numpy?

Reply:
I believe so, but then I am not sure of what exactly you're asking. How can I check?

Reply:
Actually, you took care of it when you regenerated the recipe. We needed the line `{{ pin_compatible('numpy') }}` in run.